### ISSUE TITLE
How to fix non-inherited permissions?

### ISSUE THREAD

**Issue by orion3311** [raandree/NTFSSecurity#27]
I had an issue where files were moved into folders and the user didn't have permission to change permissions on the files, hence they didn't inherit the folder's inheritable permissions. I fixed the original issue, but need to go back and correct permissions on a couple hundred files. I have a script that finds them just need the proper line to correct this, but having trouble coming up with it, and perhaps this module doesn't do this function?

Basically looking to mimic the "replace all child object permissions with inheritable permission" checkbox you see under the folder properties, but in this case for individual files.

In Explorer, if I manually add the permission (for a group) - it will now appear twice, both as explicit and inherited. I tried this using NTFSSECURITY but that didn't work - it came up only as explicit and and removed when I removed it.

**Comment by raandree** [comment]
I am not sure if I understand your issue correctly, but if you want to make sure that files are inheriting from the parent and remove all the explicitly assigned ACEs, do this:

``` PowerShell
Disable-NTFSAccessInheritance -Path D:\AL1.json -RemoveInheritedAccessRules
Enable-NTFSAccessInheritance -Path D:\AL1.json -RemoveExplicitAccessRules
```
